### DISCUSSION TITLE
能否加入自动复制填写验证码的功能

### DISCUSSION THREAD

**Discussion by o0oxzj** [gkd-kit/gkd#218] [Ideas]
如题。非常实用的功能希望大神能加入该功能

**Comment by lisonge** [comment]
https://github.com/gkd-kit/subscription/issues/new/choose

**Comment by lisonge** [comment]
你不提供快照我怎么加入？

**Comment by o0oxzj** [comment]
没有快照啊，就是加入这个功能，
--
致以问候。
Nicolas Suh
***@***.***
---
***@***.***吧！ 星期三, 01 十一月 2023, 03:48下午 +08:00 发件人 二刺螈  ***@***.*** :

**Reply by lisonge** [reply]
> 就是加入这个功能

呃呃，你说得简单，不同手机的布局节点都不一样，没有快照就不知道应该点击哪里

**Comment by o0oxzj** [comment]
快照就是我截图发给你吗？程序开发我们普通用户又不懂，只是建议嘛．不是我觉得简单．
--
致以问候。
Nicolas Suh
***@***.***
---
***@***.***吧！ 星期六, 04 十一月 2023, 00:06下午 +08:00 发件人 二刺螈  ***@***.*** :

**Reply by lisonge** [reply]
https://github.com/gkd-kit/subscription/issues/new?assignees=&labels=&projects=&template=feature_request.yml

**Comment by o0oxzj** [comment]
还有反馈一个问题，耗电量突然很高，达到25％
--
致以问候。
Nicolas Suh
***@***.***
---
***@***.***吧！ 星期六, 04 十一月 2023, 00:10下午 +08:00 发件人 二刺螈  ***@***.*** :

**Reply by lisonge** [reply]
关闭重启试试呢

**Comment by o0oxzj** [comment]
我试试，前天25，昨天19，我就直接停用了．电量受不了．还有我看了快照功能，和我想表达的增加的验证码自动填入功能好像不匹配，我是希望能加入所有的app需要填写验证码的时候，都能自动把短信收到的验证码填进去．之前用的是autotool，但是广告跳过功能比gkd差太多了，两个app同时打开又增加了额外的耗电．
--
致以问候。
Nicolas Suh
***@***.***
---
***@***.***吧！ 星期六, 04 十一月 2023, 00:14下午 +08:00 发件人 二刺螈  ***@***.*** :

**Reply by lisonge** [reply]
> 我是希望能加入所有的app需要填写验证码的时候，都能自动把短信收到的验证码填进去．

目前做不到

---

> 前天25，昨天19，我就直接停用了．电量受不了

可以先暂时关闭使用，后续我优化看看

**Comment by o0oxzj** [comment]
--
致以问候。
Nicolas Suh
***@***.***
---
***@***.***吧！ 星期六, 04 十一月 2023, 00:14下午 +08:00 发件人 二刺螈  ***@***.*** :

**Comment by o0oxzj** [comment]
好的谢谢
--
致以问候。
Nicolas Suh
***@***.***
---
***@***.***吧！ 星期六, 04 十一月 2023, 00:20下午 +08:00 发件人 二刺螈  ***@***.*** :
